android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UriBeaconDiscoveryService.java

```diff
@@ -417,6 +422,11 @@ private void updateNearbyBeaconNotification(boolean single, String url, int noti
         .setContentText(description)
         .setPriority(NOTIFICATION_PRIORITY)
         .setContentIntent(pendingIntent);
+    if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+      if (mPublicUrls.contains(url)) {
+        builder.setVisibility(NOTIFICATION_VISIBILITY);
```

Comment:
I just tried with 1, 2, and many beacons.  Everything looks correct.

I think if you set "sensitive" on the app, then it considers all the notifications sensitive regardless of what the app reports.  Make sure that is turned off and only the lock screen setting itself is set to hide sensitive notification content.